lib/open_food_network/orders_and_fulfillments_report.rb

```diff
@@ -41,9 +42,7 @@ def header
          I18n.t(:report_header_order_cycle), I18n.t(:report_header_payment_method), I18n.t(:report_header_customer_code), I18n.t(:report_header_tags),
          I18n.t(:report_header_billing_street), I18n.t(:report_header_billing_street_2), I18n.t(:report_header_billing_city), I18n.t(:report_header_billing_postcode), I18n.t(:report_header_billing_state),]
       else
-        [I18n.t(:report_header_producer), I18n.t(:report_header_product), I18n.t(:report_header_variant),
-         I18n.t(:report_header_amount), I18n.t(:report_header_curr_cost_per_unit), I18n.t(:report_header_total_cost),
-         I18n.t(:report_header_status), I18n.t(:report_header_incoming_transport)]
+        DefaultReport.new(self).header
```

Comment:
I dont understand this 5th case, there are only 4 types in the UI:
<img width="466" alt="Screenshot 2019-07-05 at 14 33 45" src="https://user-images.githubusercontent.com/1640378/60725859-0609d680-9f32-11e9-8ca8-5744af8ba8d7.png">

Reply:
Me neither, but it's the one most of the tests use...

Reply:
🙈 <- I can never remember the code for this emoji, this time I had to look it up :-)

Reply:
we should create a tech debt issue for this: make sure the different versions of the report are tested (at least the basics) and that this 5th option is removed if it is not being used. I dont think we need to fix this in this PR.

Reply:
https://github.com/openfoodfoundation/openfoodnetwork/issues/4015